src/config/pages.ts

```diff
@@ -24,25 +24,30 @@ export const PAGE_CATEGORIES: PageCategories = [
   {
     _id: 1,
     title: 'default',
+    ctitle: 'default',
   },
   {
     _id: 2,
     title: 'Stake',
+    ctitle: '抵押',
   },
   {
     _id: 3,
     title: 'Validators',
+    ctitle: '验证人',
   },
   {
     _id: 4,
     title: 'Feedback',
+    ctitle: '反馈',
   },
 ];
 
 export const PAGES_CONFIG: PagesConfig = [
   {
     category: 1,
     title: 'Overview',
+    ctitle: '总览',
```

Comment:
Same issue as above - `ctitle` should not be here. Import from translation file.